src/components/StatsGrid/StatsGridConfig.tsx

```diff
@@ -144,6 +204,16 @@ export const columnDefs = [
   {
     field: "location",
     headerName: "Location",
+    headerTooltip: (() => {
+      return [
+        htmlBold("Physical location of the node based on the ip address."),
+        "Location plays important role in determining node profitability. Nodes in the same location will share the traffic and therefore the revenue. It is best to set up nodes in locations that have potentially high traffic and limited node",
+        htmlSeparator(),
+        htmlItalic(
+          "This data is currently provided by an external service and can be inaccurate - we are working to remedy this situation."
```

Comment:
```suggestion
          "This data is currently provided by an external service and can be inaccurate. Please report any mismatch in #filecoin-saturn Slack channel."
```

Reply:
I would probably prefer people to report those via github issues https://github.com/filecoin-saturn/L1-dashboard/issues - thoughts?

Reply:
Yeah, better